Grafik/Menu.cs

```diff
@@ -91,12 +92,26 @@ public override Menu CheckMenuChoice(int userChoice)
         }
     }
 
-    public class CheckDailyShiftsSubmenu : Menu
+    public class FindEmployeeByEmailSubmenu : Menu
     {
-        public CheckDailyShiftsSubmenu()
+        public FindEmployeeByEmailSubmenu()
         {
-            Console.WriteLine("Menu 4");
+            Console.WriteLine("Wyszukaj po emailu");
+            Console.WriteLine("Wpisz Email pracownika");
+            string EmployeeEmail = (Console.ReadLine());
+
+            if (EmployeeEmail != null)
```

Comment:
warto sprawdzić czy tekst nie jest np samymi spacjami https://docs.microsoft.com/pl-pl/dotnet/api/system.string.isnullorwhitespace?view=net-5.0

Reply:
Added the check for menu 4 and 5